src/docker/orb.yml

```diff
@@ -142,7 +197,7 @@ commands:
               echo "DOCKER_PASSWORD is not set, will not be able to connect to Docker Hub" >&2
               exit 1
             fi
-      - run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+      - run: docker login -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD} ${DOCKER_SERVER:-}
```

Comment:
you should test if this new variable exists or set a default value

Reply:
the default here is to leave it empty